### PR TITLE
Qa to Master: dev mode

### PR DESCRIPTION
**Pull request by evelinec**
#37

### CODE REVIEW

**Comment by evelinec** [comment]
@Charlotte-Holt can you help review the changes made in the `README.adoc`? Thanks.

File: README.adoc [hunk 1]
```diff
@@ -111,44 +116,44 @@ Add a simple CORS configuration to the [hotspot file=0]`server.xml` file:
...
 |===
 
 Save the changes to the [hotspot file=0]`server.xml` file. The `/configurations/simple` endpoint is now ready to be
 tested with a simple CORS request.
 
-Open the [hotspot file=1]`TestCors.java` file in the `src/test/java/it/io/openliberty/guides/cors/` directory.
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-Now we will test the simple CORS configuration that you added. Add the [hotspot=25-33 file=1]`testSimpleCorsRequest` method to the `TestCors` class.
+Now we will test the simple CORS configuration that you added. Add the [hotspot=testSimpleCorsRequest file=1]`testSimpleCorsRequest` method to the `CorsIT` class.
```
Comment (Charlotte-Holt) [inline README.adoc]: Change
"Now we will test the simple CORS configuration that you added."
To
"Now, test the simple CORS configuration that you added."

File: README.adoc [hunk 2]
```diff
@@ -218,31 +218,31 @@ The preflight CORS configuration has different values than the simple CORS confi
...
-| [hotspot=19 file=0]`allowedOrigins`| Origins that are allowed to access the endpoint. The value is set to an asterisk (*) to allow requests from all origins.
-| [hotspot=20 file=0]`allowedMethods`| HTTP methods that a client is allowed to use when it makes requests to the endpoint. The value is set to `OPTIONS, DELETE`.
-| [hotspot=21 file=0]`allowCredentials`| A boolean that indicates whether the user credentials can be included in the request. The value is set to `true`.
+| [hotspot=24 file=0]`domain`|The value is set to `/configurations/preflight` because the `domain` is a different endpoint.
+| [hotspot=25 file=0]`allowedOrigins`| Origins that are allowed to access the endpoint. The value is set to an asterisk (*) to allow requests from all origins.
+| [hotspot=26 file=0]`allowedMethods`| HTTP methods that a client is allowed to use when it makes requests to the endpoint. The value is set to `OPTIONS, DELETE`.
+| [hotspot=27 file=0]`allowCredentials`| A boolean that indicates whether the user credentials can be included in the request. The value is set to `true`.
 |===
 
 The follow attributes were added:
```
Comment (Charlotte-Holt) [inline README.adoc]: Change
"The follow attributes were added:"
To
"The following attributes were added:"

File: README.adoc (hunk 1 above)
Comment (evelinec) [reply README.adoc]: Updated

File: README.adoc (hunk 2 above)
Comment (evelinec) [reply README.adoc]: Updated

**Comment by evelinec** [comment]
Changes added in #41

**Comment by evelinec** [comment]
@gkwan-ibm this is ready to publish. Thanks.

**Comment by DannyS03** [comment]
- tested/verified on `master`
- LGTM.

**Pull request merged**